db/migrate/20160714133426_rename_arbitration_defaults_to_arbitration_profiles.rb

```diff
@@ -0,0 +1,15 @@
+class RenameArbitrationDefaultsToArbitrationProfiles < ActiveRecord::Migration[5.0]
+  def up
+    rename_table :arbitration_defaults, :arbitration_profiles
+    add_column :arbitration_profiles, :name, :string
+    add_column :arbitration_profiles, :description, :text
+    add_column :arbitration_profiles, :default_profile, :boolean, :default => false
```

Comment:
Not following.  The "default" here, implies that this the default profile to use when none is provided (it's not a default value, but an indicator to use this in the absence of a specified